src/components/pages/OSPConsent/CompanyDetails.tsx

```diff
@@ -216,6 +223,12 @@ export const CompanyDetails = ({
     )
   }
 
+  const checkUniqueIdType = (type: string): string => {
+    return (
+      uniqueIdTypeText[type as keyof typeof uniqueIdTypeText] ??
+      uniqueIdTypeText.VAT_ID
+    )
+  }
```

Comment:
I suggest more compact:

```suggestion
  const checkUniqueIdType = (type: string): string =>
    uniqueIdTypeText[type as keyof typeof uniqueIdTypeText] ??
    uniqueIdTypeText.VAT_ID
```